sunshine/src/test/java/io/github/tatools/sunshine/FileConfigTest.java

```diff
@@ -13,10 +13,18 @@
 public class FileConfigTest {
 
     @Test
-    public void propertyReturnNotDefinedForUnknownKey() {
+    public void hasUnexpected() {
         MatcherAssert.assertThat(
-                new FileConfig(new ByteArrayInputStream("a =b".getBytes())).property("da"),
-                Matchers.equalTo("not defined")
+                "Unexpected property was loaded",
+                !new FileConfig(new ByteArrayInputStream("a =b".getBytes())).has("d")
+        );
+    }
+
+    @Test
```

Comment:
Should this be `@org.junit.Test`? It is not consistent in other places too... =|